self-tests.yml

```diff
@@ -819,9 +819,9 @@ buildvariants:
     display_name: Build and Push
     run_on:
       - ubuntu2204-small
-    activate: false
     tasks:
       - name: ".build"
+        priority: 100
```

Comment:
I'd envisioned we only put this priority on the staging task; this may be unnecessary for dists

Reply:
done